src/lib/loadenv.js

```diff
@@ -0,0 +1,33 @@
+// @ts-check
```

Comment:
Given how widespread this helper has become we may want to ticket creating an npm module for it!

Reply:
Sent you a slack about this.